examples/walkthrough/README.md

```diff
@@ -1,73 +1,149 @@
+### Pod Definition
+
+The simplest pod definition describes the deployment of a single container.  For example, an nginx web server pod might be defined as such:
 
 ```yaml
 apiVersion: v1
 kind: Pod
 metadata:
-  name: www
+  name: nginx
 spec:
   containers:
-    - name: nginx
-      image: nginx
+  - name: nginx
+    image: nginx
+    ports:
+    - name: http
+      containerPort: 80
 ```
 
 A pod definition is a declaration of a _desired state_.  Desired state is a very important concept in the Kubernetes model.  Many things present a desired state to the system, and it is Kubernetes' responsibility to make sure that the current state matches the desired state.  For example, when you create a Pod, you declare that you want the containers in it to be running.  If the containers happen to not be running (e.g. program failure, ...), Kubernetes will continue to (re-)create them for you in order to drive them to the desired state. This process continues until you delete the Pod.
 
 See the [design document](../../DESIGN.md) for more details.
 
+
+### Managing a Pod
+
+If you have the source checked out, the following command (executed from the root of the project) will create the pod defined above. Otherwise download [the pod definition](pod-nginx.yaml) first.
+
+```sh
+kubectl create -f examples/walkthrough/pod-nginx.yaml
+```
+
+To list all pods:
+
+```sh
+kubectl get pods
+```
+
+To view the pod's http endpoint, first determine which IP the pod was given:
+
+```sh
+kubectl get pod nginx -o=template -t={{.status.podIP}}
+```
+
+Provided your pod IPs are accessible (depending on network setup), you should be able to access the pod's http endpoint in a browser or curl on port 80.
+
+```sh
+curl http://$(kubectl get pod nginx -o=template -t={{.status.podIP}})
+```
+
+To delete the pod by name:
+
+```sh
+kubectl delete pod nginx
+```
+
+
 ### Volumes
 
-Now that's great for a static web server, but what about persistent storage?  We know that the container file system only lives as long as the container does, so we need more persistent storage.  To do this, you also declare a ```volume``` as part of your pod, and mount it into a container:
+That's great for a simple static web server, but what about persistent storage?
+
+The container file system only lives as long as the container does. So if your app's state needs to survive reboots or crashes you'll need to configure some persistent storage.  To do so, declare a ```volume``` in the pod definition, and mount it into one or more of the pod containers:
+
+Added a volume:
+```
+  volumes:
+  - name: redis-persistent-storage
+    emptyDir: {}
+```
+
+Reference the volume in our container section:
+```
+    volumeMounts:
+    # name must match the volume name below
+    - name: redis-persistent-storage
+      # mount path within the container
+      mountPath: /data/redis
+```
+
+Full definition ([source](pod-redis.yaml)):
```

Comment:
Revise to:
Result: The definition for a Redis persistent storage pod ([pod-redis.yaml](pod-redis.yaml)):